src/AspNetCore/Src/aspnetcoreconfig.cxx

```diff
@@ -48,6 +48,114 @@ ASPNETCORE_CONFIG::~ASPNETCORE_CONFIG()
 }
 
 HRESULT
+ASPNETCORE_CONFIG::CreateLogFile(
```

Comment:
I don't see why this function should belong to aspnetcoreconfig. Consider moving it to a shared location and passing in the m_struStdoutLogFile and m_fStdoutLogEnabled.

Reply:
yes. I did thought about it. Ideally, it should be a utility function